src/mame/seibu/raiden.cpp

```diff
@@ -6,18 +6,22 @@
     Seibu Raiden hardware
 
     Raiden                          (c) 1990 Seibu Kaihatsu
-    Raiden (Alternate Hardware)     (c) 1990 Seibu Kaihatsu
-    Raiden (Korean license)         (c) 1990 Seibu Kaihatsu
-    Raiden (Taiwanese license)      (c) 1990 Seibu Kaihatsu
 
     To access test mode, reset with both start buttons held.
 
-    The country/game mode byte is stored at 0xffffd in the main cpu region,
-    (that's 0x1fffe in program rom 4).
+    There are 3 configuration bytes at 0xffffb, 0xffffd and 0xfffff.
+    The byte at 0xffffb (0x1fffd in program rom 4) controls the appearance of
+    the "Winners Don't Use Drugs" screen:
+        0x00  = Normal
+        0x01  = "Winners Don't Use Drugs" screen appears on boot and between attract loops
+        0x02  = Unknown, identical to 0x00? (seen in raident and raidenk)
 
-    High nibble: Player respawn behavior when single playing
-        0x0*  = If single playing, Restart at checkpoint when every miss
-        0x8*  = Respawn instantly when every miss
+    The byte at 0xffffd (0x1fffe in program rom 4) in the main cpu region controls the
+    country/game mode:
+    High nibble: Player respawn behavior in single player
+        0x0*  = Restart at checkpoint
+        0x8*  = Respawn instantly
+        0xA*  = Unknown, identical to 0x0*? (seen in raidenk)
```

Comment:
Does it actually check for specific values, or is it just that bit 0 of 0xffffb controls the FBI screen and bit 7 of 0xffffd controls respawn behaviour?

Reply:
The lower nibble of 0xffffb is explicitly checked against 1h (code at `ef299` in raiden)
It appears that bit 7 of 0xffffd is checked, I've updated that part of the comment